tests/pipelines/multi_asic_201911.yml

```diff
@@ -0,0 +1,129 @@
+# Starter pipeline
+
+# Start with a minimal pipeline that you can customize to build and deploy your code.
+
+# Add steps that build, run tests, deploy, and more:
+
+# https://aka.ms/yaml
+
+pr:
+- master
+
+name: $(TeamProject)_$(Build.DefinitionName)_$(SourceBranchName)_$(Date:yyyyMMdd)$(Rev:.r)
+
+resources:
+  repositories:
+  - repository: sonic-mgmt
+    type: github
+    name: Azure/sonic-mgmt
+    endpoint: build
+
+stages:
+- stage: Test
+  pool: sonictest
+  variables:
+  - name: dut
+    value: vlab-08
```

Comment:
These hardcoded variables can be made as input parameters with default value. That will help changing the parameters if testbed.csv or any inventory changes.

Reply:
This can be addressed in later PR

Reply:
Modified as per suggestion.